src/App.js

```diff
@@ -13,9 +13,9 @@ function App() {
   return (
     <div>
       <h1>Todo List</h1>
-      <AddTodoForm onAdd={onAddTodo} />
+      <AddTodoForm onAddTodo={onAddTodo} />
```

Comment:
you didn't define the function "onAddTodo", instead you should use setNewTodo to change the state "newTodo"